### PR TITLE
Revert the removal of an option to block autoplay

### LINKED ISSUE #791: Revert the removal of an option to block autoplay from Chrome
### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

No, but I believe a quick search in chromium git will avail for a quick undoing of the change.

JS based solutions are available https://github.com/Eloston/disable-html5-autoplay

### Describe the solution you would like

Revert to autoplay blocking the way it was done in chrome 7x.x, where autoplay block was blocking all videos.

Android Chrome 82.x+ will **still play** videos even if they are autoplay blocked if they pass some obscure "engagement score:" https://www.theregister.com/2020/09/08/google_chrome_autoplay/

### Describe alternatives you have considered

Reverting to older versions works

### PR DESCRIPTION
**Pull request by uazo**
fix #791

### CODE REVIEW

**Review by csagan5 — CHANGES_REQUESTED** [review]
Please submit the patch without removing anything from it (`index` lines etc); it is currently corrupt.

```
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-hdpi/settings_autoplay.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-hdpi/settings_autoplay.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-hdpi/settings_media.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-hdpi/settings_media.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-mdpi/settings_autoplay.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-mdpi/settings_autoplay.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-mdpi/settings_media.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-mdpi/settings_media.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xhdpi/settings_autoplay.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xhdpi/settings_autoplay.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xhdpi/settings_media.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xhdpi/settings_media.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xxhdpi/settings_autoplay.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xxhdpi/settings_autoplay.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xxhdpi/settings_media.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xxhdpi/settings_media.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xxxhdpi/settings_autoplay.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xxxhdpi/settings_autoplay.png: patch does not apply
error: cannot apply binary patch to 'components/browser_ui/site_settings/android/java/res/drawable-xxxhdpi/settings_media.png' without full index line
error: components/browser_ui/site_settings/android/java/res/drawable-xxxhdpi/settings_media.png: patch does not apply
```

File: build/bromite_patches_list.txt [hunk 1]
```diff
@@ -150,4 +150,5 @@ Remove-weblayer-dependency-on-Play-Services.patch
 Timezone-customization.patch
 Move-some-account-settings-back-to-privacy-settings.patch
 Add-a-flag-to-always-view-desktop-site-for-all-websites.patch
+Revert-removal-option-to-block-autoplay.patch
```
[the remark this answers is not in the document]
Comment (csagan5) [reply build/bromite_patches_list.txt:153]: I made a commit to fix this.

**Comment by csagan5** [comment]
@uazo I have also fixed the corrupt patch; please see my commit.

**Review by csagan5 — APPROVED** [review]
There are a few things to fix, but I will fix them before next release myself:
* the `Media` menu seems to be useless, since it points to the same site settings; I might have misunderstood its importance; I will remove it
* default should be `Autoplay disabled`
* it does not show in the page info site settings (if I cannot fix this for this release, I will leave it to you on next release)

File: build/bromite_patches_list.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (uazo) [reply build/bromite_patches_list.txt:153]: sorry, translation error, in Italian "can" is "puoi" (if you want), simply I was trusting that you was be pleased.
no problem, next time I'll put it on the bottom.

**Comment by uazo** [comment]
>- default should be Autoplay disabled
> - it does not show in the page info site settings (if I cannot fix this for this release, I will leave it to you on next release)

yes, I thought we discussed it first
I would also say that we should understand if "media engamenent" is useful for privacy

**Comment by csagan5** [comment]
I do not follow you.

I have fixed all of the above, the only issue left is that it does not show on the page info when clicking on the URL bar icon. I am trying to fix it before release.

**Comment by uazo** [comment]
> I do not follow you.

I'm referring to "some obscure engagement score"

**Comment by csagan5** [comment]
> I'm referring to "some obscure engagement score"

Ok, but we are [discussing that in the issue](https://github.com/bromite/bromite/issues/791#issuecomment-723547781) not here.

**Pull request merged**